dev-tools/gen-windows-sys-binding/src/main.rs

```diff
@@ -49,16 +51,39 @@ fn main() -> io::Result<()> {
     // Generate bindings.
     windows_bindgen::bindgen(&args).expect("running bindgen failed");
 
-    let mut bindings = String::new();
-    fs::File::open(temp_file.path())
-        .expect("failed to open temp windows_sys.rs")
-        .read_to_string(&mut bindings)
-        .expect("failed to read temp windows_sys.rs");
+    let bindings = fs::read_to_string(temp_file.path())
+        .expect("failed to read temp windows_sys.rs")
+        .replace(
+            "windows_targets::link!",
+            "super::windows_targets::link_macro!",
+        );
```

Comment:
Rather than rewriting the macro line, why not just add `use super::windows_targets;`? Then in windows_targets you'd just need `pub(crate) use link_macro as link;`.

Reply:
Hmmm let me have a try, I didn't think of exporting it with rename.

Reply:
Thank you!

Updated the code